### DISCUSSION TITLE
RSL new bug

### DISCUSSION THREAD

**Discussion by WiseDragonGR** [FarbstoffRSL/RSL-Helper#491] [General]
It happens last few days with new version. Tested on 3 different computers. 
Autoclicker at some point stops working when run out of energy and asks to buy energy, so Autoclicker stops working instead of waiting till energy reffils after few minutes *as it used to

![RSL Bug](https://user-images.githubusercontent.com/102595214/233442119-a3c714cd-28b5-4132-9823-9f0015332590.jpg)

**Comment by crysty4noone** [comment]
Same problem. What is the solution?

**Comment by nberanger** [comment]
I have noticed this same issue was well. Was hoping that latest update would fix it, but it still occurs.
